portfolio/src/main/webapp/script.js

```diff
@@ -48,3 +48,40 @@ function openContent(evt, page) {
     footerElement.classList.add("minimiseFooter");
   }
 }
+
+/**
+ * Fetches a my name from the server and adds it to the DOM.
+ */
+function getName() {
+  console.log('Fetching my name.');
```

Comment:
Is this for debug or do you need it in code?

Reply:
Same for the below 2.

Reply:
Fixed!